templates/base.html

```diff
@@ -38,7 +38,8 @@
     <link rel="icon" type="image/png" href="{{ url_for('static', filename='img/favicon/favicon-16x16.png') }}" sizes="16x16">
     <link rel="manifest" href="{{ url_for('static', filename='img/favicon/manifest.json') }}">
     <link rel="mask-icon" href="{{ url_for('static', filename='img/favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
-    <link rel="shortcut icon" href="{{ url_for('static', filename='img/favicon/favicon.ico') }}">
+    <!-- favicon changes depending on server type -->
+    <link rel="shortcut icon" href="{{ url_for('static', filename=config.FAVICON.get(config.SERVER_TYPE))  }}">
```

Comment:
Rather than a dictionary with all possible favicon filenames, we could use a string that gets set per platform in settings_local.py

For example, we use this with SERVER_TYPE as shown below:
```
SERVER_TYPE = 'Private QA'
```

Similarly, we could change this value per platform in the config:

```
FAVICON_FILENAME = 'favicon_qa.ico'
```

The resulting HTML would then be:

```html
<link rel="shortcut icon" href="{{ url_for('static', filename=config.FAVICON_FILENAME)  }}">
```

Reply:
Hi Kory, I think this is similar to a change I made with my most recent commit - also using server type but instead having platform-specific folders containing the appropriate favicons. Thanks!

Reply:
I think I misunderstood your comment yesterday - making the fix you suggested now.

Reply:
Additionally, please set a default favicon to handle the case of a missing config value.

```
<link rel="shortcut icon" href="{{ url_for('static', filename=config.get(FAVICON_FILENAME, 'favicon.ico'))  }}">
```

This should be tested by removing FAVICON_FILENAME from settings_local.py and ensuring the default icon displays correctly. Thank you.